coderd/users.go

```diff
@@ -1272,6 +1277,48 @@ func (api *API) CreateUser(ctx context.Context, store database.Store, req Create
 		}
 		return nil
 	}, nil)
+	if err == nil && !req.SkipNotifications {
+		// Notify user admins
+		// Get all users with user admin permission including owners
+		var owners, userAdmins []database.GetUsersRow
+		var eg errgroup.Group
+		eg.Go(func() error {
+			var err error
+			owners, err = api.Database.GetUsers(ctx, database.GetUsersParams{
+				RbacRole: []string{codersdk.RoleOwner},
```

Comment:
Can you not use multiple `RbacRoles` here, or does that find the intersection of those roles?

Reply:
Yes, it is an intersection.